Makefile

```diff
@@ -5,7 +5,7 @@ export PLUGINPATH:="connect-plugins"
 # fix for ubuntu sid sometimes linking /bin/sh to /bin/dash
 SHELL:=/bin/bash
 
-all: install irc transform
+all: install irc transform gencerts
```

Comment:
This. Is. Awesome.

Reply:
Yep, it's trivial to add it to whatever new setup script comes into the fray. Happy to do so if this doesn't get merged before that happens.